src/styles/main.scss

```diff
@@ -1,3 +1,56 @@
-body {
-  margin: 0;
+.main {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  font-family: Arial, sans-serif;
+  margin: 9px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: auto;
+  width: 721px;
```

Comment:
Avoid such a hard-coded values.
This is like a magic number, it is not clear why is it set to 721px and if you change for instance dimensions of calendar-day things will break up.
You should calculate the width basing on your variables ;)